Decred Wallet/en.lproj/Localizable.strings

```diff
@@ -383,3 +379,12 @@
 "errorMsg" = "Error Message";
 "info" = "Info";
 "delete" = "Delete";
+
+
+/* Other Text */
```

Comment:
This needs to be done for other language files as well.

Reply:
resolved in 37e2453